react-native-libraries.json

```diff
@@ -11824,21 +11824,54 @@
     "newArchitecture": true
   },
   {
-    "githubUrl": "https://github.com/gabriel-logan/mobile-native-barcode-generator",
-    "npmPkg": "mobile-native-barcode-generator",
+    "githubUrl": "https://github.com/Multiform-Validator/typescript-javascript",
+    "npmPkg": "multiform-validator",
     "examples": [
-      "https://github.com/gabriel-logan/mobile-native-barcode-generator/tree/main/example",
-      "https://github.com/gabriel-logan/mobile-native-barcode-generator/tree/main/docs"
+      "https://multiformvalidator.netlify.app/documentation/js"
     ],
-    "images": ["https://raw.githubusercontent.com/gabriel-logan/mobile-native-barcode-generator/main/docs/assets/example2.png"],
-    "android": true
+    "images": ["https://raw.githubusercontent.com/Multiform-Validator/docs/main/src/app/opengraph-image.png"],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "newArchitecture": true
   },
   {
-    "githubUrl": "https://github.com/gabriel-logan/expo-barcode",
-    "npmPkg": "expo-barcode",
+    "githubUrl": "https://github.com/gabriel-logan/Azure-translator-code",
+    "npmPkg": "azure-translator-code",
     "examples": [
-      "https://github.com/gabriel-logan/expo-barcode/tree/main/example"
+      "https://azuretranslatorcode.vercel.app/"
     ],
-    "android": true
+    "ios": true,
```

Comment:
We are still removing the packages added in earlier PR from today.

Reply:
Oh ok.

Reply:
Do I need do something ?

Reply:
Yup, you need to rebase the PR and resolve conflicts, but at this point it might be easier, just to re-create the changeset as a new PR which we can merge in.

Reply:
ok